ocaml/xapi/xapi.ml

```diff
@@ -680,6 +680,14 @@ let set_stunnel_legacy_db ~__context () =
     Xapi_host.set_stunnel_legacy ~__context legacy_db
   end
 
+(* Consult database to see if this host has any interfaces on networks
```

Comment:
We don't do this right now, or did you add this comment for future work?

Reply:
The latter, hence the "TODO" part of the comment.